modules/ROOT/pages/additional-information/kb-documents/fail2ban-protect-login.adoc

```diff
@@ -28,27 +28,32 @@ The log for a failed login attempt looks like this and consists of two consecuti
 {"level":"info","service":"proxy","proto":"HTTP/1.0","request-id":"blabla","remote-addr":"123.123.123.123","method":"POST","status":204,"path":"/signin/v1/identifier/_/logon","duration":135.139963,"bytes":0,"time":"2023-03-20T19:26:04.727076622Z","message":"access-log"}
 ----
 
-Note that the IP address of the source (`123.123.123.123`) is not in the line where the failed login attempt (`invalid credentials`) is detected.
+* The first log entry is used to detect the `invalid credentials` message only but it does not contain the correct IP address.
+* The other related log entry contains the needed IP address for further processing.
```

Comment:
Do we have to mention that ocis should be started with `info` log level? because the line containing IP address is only available when using the info log level.

Reply:
added a note, see commit: [add minimum loglevel](https://github.com/owncloud/docs-ocis/pull/569/commits/aa039ade9609358b0339fd58d7f7706643086fd9)